controllers/admin/AdminCmsContentController.php

```diff
@@ -75,14 +78,6 @@ public static function getCurrentCMSCategory()
         return self::$category;
     }
 
-    public function viewAccess($disable = false)
```

Comment:
Watch out that this method is not used anywhere else. If it is, please, just deprecate the method for now